src/plugins/geomap/demo/config-fr.js

```diff
@@ -108,7 +117,7 @@ var wet_boew_geomap = {
 						"p91tLJi/b3ppudc1jH2z3b1vhp4Ehx3WrlkOKQvNYp5STfUKoi1/Jv4WUualAICCL5ax4YTUtMLFSzmriL1z5gWQ67XYNAAAAAElFTkSuQmCC",
 						fillOpacity: "1",
 						graphicWidth: "25",
-						name: "Mines de métaux"
+						name: "Metal mines"
```

Comment:
Do we need to use the English name? Can we keep the French name?

Reply:
Fixed in 27caeb5aa2b3fef1718e261c90ebec24dba2c448